src/helpers/encodeParams.js

```diff
@@ -4,9 +4,10 @@ const serializeParam = (prefix, param, value) => {
   if (typeof (value) === 'object') {
     return Object.keys(value)
       .map(key => serializeParam(prefixedParam, parseKey(key), value[key]))
+      .filter(serializedParam => !!serializedParam)
       .join('&');
   } else {
-    return [prefixedParam, value].join('=');
+    return value && [prefixedParam, value].join('=');
```

Comment:
To tell you the truth it differs for everyone :)

If it's a consistent separator I will always use `join()`

```
['query', param].join('=')
[param1, param2].join(',')
['path', id 'otherpath'].join('/')
```

Otherwise, when there are different separators use `${}`

```
`http://${domain}/path/path?${query}`
```